src/orders/components/OrderFulfillPage/utils.ts

```diff
@@ -0,0 +1,32 @@
+import { OrderFulfillDataQuery } from "@saleor/graphql";
+
+export const getDefaultFulfillmentValue = (
+  line: OrderFulfillDataQuery["order"]["lines"][0],
```

Comment:
avoid deeply nested types, extract to fragment and then import that fragment's type

example: 
```ts
// src/fragments/errors.ts:266
export const voucherBulkDeleteError = gql`
  fragment VoucherBulkDeleteError on DiscountError {
    code
    field
    message
  }
`;
```
then import from types.generated.ts

```ts
export type VoucherBulkDeleteErrorFragment = { __typename: 'DiscountError', code: DiscountErrorCode, field: string | null, message: string | null };
```